bin/pygac-convert-patmosx-coefficients

```diff
@@ -0,0 +1,274 @@
+        r'\s*(?P<ch5_Ns>[eE0-9\.-]+),?\s*(?P<ch5_b0>[eE0-9\.-]+),?\s*(?P<ch5_b1>[eE0-9\.-]+),?\s*(?P<ch5_b2>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<nu_3b>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<a1_3b>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<a2_3b>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<nu_4>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<a1_4>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<a2_4>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<nu_5>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<a1_5>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<a2_5>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch1_dark_count>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch2_dark_count>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch3a_dark_count>[eE0-9\.-]+)[^\n]*\n'
+        r'(?:[a-z]+[^\n]*\n)?'
+        r'\s*(?P<ch1_low_gain_S0>[eE0-9\.-]+)\s*(?P<ch1_low_gain_S1>[eE0-9\.-]+)\s*(?P<ch1_low_gain_S2>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch1_high_gain_S0>[eE0-9\.-]+)\s*(?P<ch1_high_gain_S1>[eE0-9\.-]+)\s*(?P<ch1_high_gain_S2>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch2_low_gain_S0>[eE0-9\.-]+)\s*(?P<ch2_low_gain_S1>[eE0-9\.-]+)\s*(?P<ch2_low_gain_S2>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch2_high_gain_S0>[eE0-9\.-]+)\s*(?P<ch2_high_gain_S1>[eE0-9\.-]+)\s*(?P<ch2_high_gain_S2>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch3a_low_gain_S0>[eE0-9\.-]+)\s*(?P<ch3a_low_gain_S1>[eE0-9\.-]+)\s*(?P<ch3a_low_gain_S2>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch3a_high_gain_S0>[eE0-9\.-]+)\s*(?P<ch3a_high_gain_S1>[eE0-9\.-]+)\s*(?P<ch3a_high_gain_S2>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<date_of_launch>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch1_gain_switches_count>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch2_gain_switches_count>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<ch3a_gain_switches_count>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<PRT1_0>[eE0-9\.-]+)\s*(?P<PRT1_1>[eE0-9\.-]+)\s*(?P<PRT1_2>[eE0-9\.-]+)\s*(?P<PRT1_3>[eE0-9\.-]+)\s*(?P<PRT1_4>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<PRT2_0>[eE0-9\.-]+)\s*(?P<PRT2_1>[eE0-9\.-]+)\s*(?P<PRT2_2>[eE0-9\.-]+)\s*(?P<PRT2_3>[eE0-9\.-]+)\s*(?P<PRT2_4>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<PRT3_0>[eE0-9\.-]+)\s*(?P<PRT3_1>[eE0-9\.-]+)\s*(?P<PRT3_2>[eE0-9\.-]+)\s*(?P<PRT3_3>[eE0-9\.-]+)\s*(?P<PRT3_4>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<PRT4_0>[eE0-9\.-]+)\s*(?P<PRT4_1>[eE0-9\.-]+)\s*(?P<PRT4_2>[eE0-9\.-]+)\s*(?P<PRT4_3>[eE0-9\.-]+)\s*(?P<PRT4_4>[eE0-9\.-]+)[^\n]*\n'
+        r'\s*(?P<PRT_weight_1>[eE0-9\.-]+)\s*(?P<PRT_weight_2>[eE0-9\.-]+)\s*(?P<PRT_weight_3>[eE0-9\.-]+)\s*(?P<PRT_weight_4>[eE0-9\.-]+)[^\n]*\n'
+        r'(?:\s*(?P<sst_mask_day_0>[eE0-9\.-]+),\s*(?P<sst_mask_day_1>[eE0-9\.-]+),\s*(?P<sst_mask_day_2>[eE0-9\.-]+),\s*(?P<sst_mask_day_3>[eE0-9\.-]+)[^\n]*\n)?'
+        r'(?:\s*(?P<sst_mask_night_0>[eE0-9\.-]+),\s*(?P<sst_mask_night_1>[eE0-9\.-]+),\s*(?P<sst_mask_night_2>[eE0-9\.-]+),\s*(?P<sst_mask_night_3>[eE0-9\.-]+)[^\n]*\n)?'
+        r'(?:\![^v][^\n]*\n)*'
+        r'(?:\!(?P<version>v\w+))?'
+    )
+
+    def __init__(self, tarball):
+        self.tarball = tarball
```

Comment:
The PATMOS-x webpage provides individual coefficient files, not tarballs. Would it make sense to pass a list of filenames here instead?

Reply:
Actually, they provide the tarballs in the section "AVHRR instrument files with older calibration coefficients". The name of this section is a little bit misleading, because it says *Older Version Archive*, but it also contains the tarball for the presented individual files `2017 (SBAF) instrument files`.

Reply:
Ok, thanks for clarifying!